zeppelin-interpreter-parent/pom.xml

```diff
@@ -52,6 +53,10 @@
           <groupId>io.atomix</groupId>
           <artifactId>*</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
```

Comment:
does this have real effects?

Reply:
Definitely, I changed the image in the pull request description and labeled the German columns additionally.

Reply:
Before PR
```
root@f8d6a0394383:/# du -sh /opt/zeppelin/interpreter/*
83M	/opt/zeppelin/interpreter/alluxio
50M	/opt/zeppelin/interpreter/angular
57M	/opt/zeppelin/interpreter/bigquery
87M	/opt/zeppelin/interpreter/cassandra
75M	/opt/zeppelin/interpreter/elasticsearch
54M	/opt/zeppelin/interpreter/file
101M	/opt/zeppelin/interpreter/flink
36M	/opt/zeppelin/interpreter/flink-cmd
57M	/opt/zeppelin/interpreter/groovy
64M	/opt/zeppelin/interpreter/hbase
55M	/opt/zeppelin/interpreter/influxdb
51M	/opt/zeppelin/interpreter/java
24M	/opt/zeppelin/interpreter/jdbc
69M	/opt/zeppelin/interpreter/jupyter
57M	/opt/zeppelin/interpreter/livy
72M	/opt/zeppelin/interpreter/md
51M	/opt/zeppelin/interpreter/mongodb
57M	/opt/zeppelin/interpreter/neo4j
81M	/opt/zeppelin/interpreter/python
202M	/opt/zeppelin/interpreter/r
69M	/opt/zeppelin/interpreter/sh
88M	/opt/zeppelin/interpreter/spark
65M	/opt/zeppelin/interpreter/spark-submit
65M	/opt/zeppelin/interpreter/sparql
69M	/opt/zeppelin/interpreter/zeppelin-interpreter-shaded-0.12.0-SNAPSHOT.jar
```
After PR
```
root@e052296353cb:/# du -sh /opt/zeppelin/interpreter/*
83M	/opt/zeppelin/interpreter/alluxio
484K	/opt/zeppelin/interpreter/angular
7.4M	/opt/zeppelin/interpreter/bigquery
87M	/opt/zeppelin/interpreter/cassandra
25M	/opt/zeppelin/interpreter/elasticsearch
4.3M	/opt/zeppelin/interpreter/file
52M	/opt/zeppelin/interpreter/flink
36M	/opt/zeppelin/interpreter/flink-cmd
7.3M	/opt/zeppelin/interpreter/groovy
14M	/opt/zeppelin/interpreter/hbase
4.8M	/opt/zeppelin/interpreter/influxdb
780K	/opt/zeppelin/interpreter/java
24M	/opt/zeppelin/interpreter/jdbc
19M	/opt/zeppelin/interpreter/jupyter
7.5M	/opt/zeppelin/interpreter/livy
22M	/opt/zeppelin/interpreter/md
1.6M	/opt/zeppelin/interpreter/mongodb
7.6M	/opt/zeppelin/interpreter/neo4j
31M	/opt/zeppelin/interpreter/python
202M	/opt/zeppelin/interpreter/r
20M	/opt/zeppelin/interpreter/sh
88M	/opt/zeppelin/interpreter/spark
16M	/opt/zeppelin/interpreter/spark-submit
16M	/opt/zeppelin/interpreter/sparql
69M	/opt/zeppelin/interpreter/zeppelin-interpreter-shaded-0.12.0-SNAPSHOT.jar
```

Reply:
@Reamer I do not mean the whole PR, just this

```patch
+ <exclusion>
+   <groupId>org.apache.hadoop</groupId>
+   <artifactId>*</artifactId>
+ </exclusion>
``` 

```
./mvnw clean package -Pbuild-distr -DskipTests
```

before
```
du -hd1 interpreter 
7.5M    interpreter/neo4j
 15M    interpreter/sparql
 83M    interpreter/alluxio
4.2M    interpreter/file
 31M    interpreter/python
 30M    interpreter/flink
154M    interpreter/r
476K    interpreter/angular
772K    interpreter/java
7.9M    interpreter/bigquery
1.6M    interpreter/mongodb
2.7M    interpreter/jdbc
4.7M    interpreter/influxdb
 20M    interpreter/sh
 15M    interpreter/spark-submit
7.5M    interpreter/livy
 39M    interpreter/cassandra
 22M    interpreter/md
 14M    interpreter/hbase
7.2M    interpreter/groovy
 16M    interpreter/flink-cmd
 19M    interpreter/jupyter
 25M    interpreter/elasticsearch
 38M    interpreter/spark
585M    interpreter
```

after
```
du -hd1 interpreter                           
7.5M    interpreter/neo4j
 15M    interpreter/sparql
 96M    interpreter/alluxio
4.2M    interpreter/file
 31M    interpreter/python
 30M    interpreter/flink
166M    interpreter/r
476K    interpreter/angular
772K    interpreter/java
7.9M    interpreter/bigquery
1.6M    interpreter/mongodb
2.7M    interpreter/jdbc
4.7M    interpreter/influxdb
 20M    interpreter/sh
 15M    interpreter/spark-submit
7.5M    interpreter/livy
 39M    interpreter/cassandra
 22M    interpreter/md
 14M    interpreter/hbase
7.2M    interpreter/groovy
 16M    interpreter/flink-cmd
 19M    interpreter/jupyter
 26M    interpreter/elasticsearch
 38M    interpreter/spark
612M    interpreter
```

Reply:
@pan3793 
Yes, this is also necessary when working with the profile `-Pinclude-hadoop`.